cloud/kubernetes/helm/yugabyte/templates/_helpers.tpl

```diff
@@ -38,3 +38,24 @@ Create chart name and version as used by the chart label.
 {{- define "yugabyte.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+  Get YugaByte fs data directories
+*/}}
+{{- define "yugabyte.fs_data_dirs" -}}
+{{range $index := until (int (.count))}}{{if ne $index 0}},{{end}}/mnt/disk{{ $index }}{{end}}
+{{- end -}}
+
+{{/*
+  Get YugaByte master addresses
+*/}}
+{{- define "yugabyte.master_addresses" -}}
+{{range $index := until (int (.Values.replicas.master))}}{{if ne $index 0}},{{end}}yb-master-{{ $index }}.yb-masters.$(NAMESPACE):7100{{end}}
```

Comment:
Missing .svc and .domainName here -- should be specified to avoid DNS thrashing

Reply:
sure let me fix this as well.